data_model/src/name.rs

```diff
@@ -88,6 +90,7 @@ impl AsRef<str> for Name {
     }
 }
 
+#[cfg_attr(all(feature = "ffi_export", not(feature = "ffi_import")), ffi_export)]
```

Comment:
If possible I would encourage you to add the `ffi_import` as well. The issue wasn't created with it in mind because `ffi_import` didn't exist at that time